alg/teca_2d_component_area.h

```diff
@@ -44,6 +44,22 @@ stored in the output dataset metadata in the following keys:
     background_id - the label used for cells outside of the segmentation,
                     i.e. the background. This can be used to skip processing
                     of the background when desirable.
+
+@rst
+.. table:: Algorithm Properties
+
+    +----------------------------+------------------------------------------------------------+
+    |         Property           |                      Description                           |
+    +============================+============================================================+
+    | component_variable         | the component variable name containing the region labels.  |
+    +----------------------------+------------------------------------------------------------+
+    | contiguous_component_ids   | the label used for cells inside the segmentation.          |
+    +----------------------------+------------------------------------------------------------+
+    | background_id              | | the label used for cells outside of the segmentation,    |
+    |                            | | i.e. the background. This can be used to skip processing |
+    |                            | | of the background when desirable.                        |
+    +----------------------------+------------------------------------------------------------+
+@endrst
```

Comment:
I'm not sold on the idea of using rst tables for this. What's the reason? Was it to test with Breathe?

Also, use \@ref component_variable etc to make a link to the property's documentation